torchvision/io/image.py

```diff
@@ -166,7 +167,11 @@ def decode_jpeg(input: torch.Tensor, mode: ImageReadMode = ImageReadMode.UNCHANG
     Returns:
         output (Tensor[image_channels, image_height, image_width])
     """
-    output = torch.ops.image.decode_jpeg(input, mode.value)
+    device = torch.device(device)
+    if device.type == 'cuda':
```

Comment:
Should the dispatching be done here or in C++? How will it interact with torchscript?

Reply:
I was thinking about making it dispatch on C++ directly, but this should also work with torchscript I believe